discord/monetization.py

```diff
@@ -173,4 +173,4 @@ async def delete(self) -> None:
         HTTPException
             Deleting the entitlement failed.
         """
-        await self._state.http.delete_test_entitlement(self.id)
+        await self._state.http.delete_test_entitlement(self.application_id, self.id)
```

Comment:
While this is an appropriate change, also beyond the scope of the PR.

Reply:
should i make another PR then?

Reply:
Yes, this would be a suitable fix pr on its own